services.sh

```diff
@@ -4,6 +4,8 @@
 : ${DRUID_HOST:="localhost"}
 : ${DRUID_PORT:="8082"}
 
+set -ex
```

Comment:
This makes a mess of the usage statement when you run `services.sh` with no or incorrect arguments.

Reply:
Changed to `set -e`